Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -489,6 +500,10 @@ void ProgramShaderCache::Shutdown()
 			u8 *binary = data + sizeof(GLenum);
 			GLenum *prog_format = (GLenum*)data;
 			glGetProgramBinary(entry.second.shader.glprogid, binary_size, nullptr, prog_format, binary);
+			if (glGetError() != GL_NO_ERROR)
+			{
+				continue;
```

Comment:
This will leak the memory allocated for data. Mind changing my old broken code into a vector?